.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,12 @@ Fixes #0.
 
 **Other considerations**
 
+**Testing**
+- [ ] Mobile
+- [ ] Tablet
+- [ ] Desktop
+- [ ] _List all the testing tasks that applies to your fix and help peers to review your work._
+
```

Comment:
Love this. Thanks Meli

While we're already updating the template, would you mind including a line under "Demo links" that says something like" 
_Please include a link to a demo store that includes preconfigured sections to allow reviewers to easily test the features you are working on._